### PR TITLE
layout stars

### PR DESCRIPTION
**Pull request by AndriiYelieva**
- [DEMO LINK](https://AndriiYelieva.github.io/layout_stars/)
- [TEST REPORT LINK](https://AndriiYelieva.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  background: url(./images/star-active.svg) no-repeat center;
+}
+
+.stars--4 :nth-child(-n+4) {
+  background: url(./images/star-active.svg) no-repeat center;
+}
+
+.stars--5 :nth-child(-n+5) {
+  background: url(./images/star-active.svg) no-repeat center;
+}
```
Comment (OlEzhA0) [inline src/style.css:38]: you can use next syntax when you have similar styles

```
.class1,
.class2,
.class3 {
  color: #000;
}
```

**Review by OlEzhA0 — CHANGES_REQUESTED** [review]
hey! check my comment please

File: src/index.html [hunk 2]
```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container stars--0">
```
Comment (Anastasiia-Svintsova) [inline src/index.html:11]: Fix it everywhere
image

```suggestion
    <div class="stars stars--0">
```

File: src/index.html [hunk 3]
```diff
@@ -8,6 +8,47 @@
...
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="container stars--1">
```
Comment (Anastasiia-Svintsova) [inline src/index.html:18]:
```suggestion
    </div>
    
    <div class="container stars--1">
```

File: src/index.html [hunk 4]
```diff
@@ -8,6 +8,47 @@
...
+    <div class="container stars--5">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
   </body>
-</html>
+</html>
```
Comment (Anastasiia-Svintsova) [inline src/index.html:54]: Add an empty line at the end of the file
```suggestion
</html>

```

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
Good! Check my comments

**Comment by AndriiYelieva** [comment]
Hi, I'm throwing a new fixed file, but I don't see if I replaced it

File: src/index.html (hunk 2 above)
Comment (mykhalenych) [reply src/index.html:11]: Didn't fix it

**Review by mykhalenych — CHANGES_REQUESTED** [review]
Comments aren't fixed, if you have any questions, ask in the chat

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+.stars :not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: url(images/star.svg) no-repeat center;
+  border-radius: 0.5px;
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:17]: do you need `border-radius` here?
also do not use non-integer values for px

**Review by Bohdan-Maksymiuk — APPROVED** [review]
Good job 👍

**Pull request closed without merging**